crates/biome_grit_formatter/Cargo.toml

```
@@ -20,6 +20,7 @@ biome_rowan       = { workspace = true }
 [dev-dependencies]
 biome_configuration  = { path = "../biome_configuration" }
 biome_formatter_test = { path = "../biome_formatter_test" }
+biome_fs             = { workspace = true }
```

```suggestion
biome_fs             = { path = "../biome_fs" }
```

This is really important